ruma-events/src/room/avatar.rs

```diff
@@ -22,7 +22,13 @@ pub struct AvatarEventContent {
 
     /// Information about the avatar thumbnail image.
     /// URL of the avatar image.
+    #[cfg(not(feature = "unstable-synapse-quirks"))]
     pub url: String,
+
+    /// Information about the avatar thumbnail image.
```

Comment:
I think this first line of the field's docs is redundant. Can you remove it (from both `cfg`s)?